contracts/UniswapV3Pool.sol

```diff
@@ -158,20 +154,13 @@ contract UniswapV3Pool is IUniswapV3Pool, NoDelegateCall {
         return abi.decode(data, (uint256));
     }
 
-    /// @inheritdoc IUniswapV3PoolDerivedState
-    function secondsInside(int24 tickLower, int24 tickUpper) external view override noDelegateCall returns (uint32) {
```

Comment:
the expectation is that callers will just do this themselves now?

Reply:
and same with e.g. `secondsPerLiquidityInsideX128`?

Reply:
Get money

Reply:
handled in the other pr